<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CoverOptionsPage</class>
 <widget class="QWidget" name="CoverOptionsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>632</width>
    <height>560</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="rename_files">
     <property name="title">
      <string>Location</string>
     </property>
     <layout class="QVBoxLayout">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="margin">
       <number>9</number>
      </property>
      <item>
       <widget class="QCheckBox" name="save_images_to_tags">
        <property name="text">
         <string>Embed cover images into tags</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cb_embed_front_only">
        <property name="text">
         <string>Only embed a front image</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="save_images_to_files">
        <property name="text">
         <string>Save cover images as separate files</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Use the following file name for images:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="cover_image_filename"/>
      </item>
      <item>
       <widget class="QCheckBox" name="save_images_overwrite">
        <property name="text">
         <string>Overwrite the file if it already exists</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="ca_providers_groupbox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Cover Art Providers</string>
     </property>
     <layout class="QVBoxLayout" name="ca_providers_layout">
      <item>
       <widget class="QTabWidget" name="tab_cover_art_providers">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="tabPosition">
         <enum>QTabWidget::North</enum>
        </property>
        <property name="currentIndex">
         <number>-1</number>
        </property>
        <property name="movable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="reordertabs">
        <item>
         <widget class="QLabel" name="movetabnote">
          <property name="font">
           <font>
            <pointsize>8</pointsize>
            <italic>true</italic>
           </font>
          </property>
          <property name="text">
           <string>Providers will be run in the order of the tabs, starting from left. Use those buttons or drag'n'drop to change order.</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="moveleft">
          <property name="text">
           <string>Move to left</string>
          </property>
          <property name="arrowType">
           <enum>Qt::LeftArrow</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="moveright">
          <property name="text">
           <string>Move to right</string>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonIconOnly</enum>
          </property>
          <property name="arrowType">
           <enum>Qt::RightArrow</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>save_images_to_tags</tabstop>
  <tabstop>cb_embed_front_only</tabstop>
  <tabstop>save_images_to_files</tabstop>
  <tabstop>cover_image_filename</tabstop>
  <tabstop>save_images_overwrite</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>save_images_to_tags</sender>
   <signal>clicked(bool)</signal>
   <receiver>cb_embed_front_only</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>266</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>266</x>
     <y>67</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
